src/graphql/apollo/client.ts

```diff
@@ -1,11 +1,15 @@
-import ApolloClient from 'apollo-boost';
-import defaults from './defaults';
+import { ApolloOfflineClient } from 'offix-client';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { HttpLink } from 'apollo-link-http';
 
-const client = new ApolloClient({
-  uri: process.env.REACT_APP_BACKEND_HOST || 'http://localhost:4000/',
-  clientState: {
-    defaults,
-  },
-});
+const config = {
+  link: new HttpLink({
+    uri: process.env.REACT_APP_BACKEND_HOST || 'http://localhost:4000/',
+  }),
+  cache: new InMemoryCache(),
+};
```

Comment:
what are these updated config values doing?

Reply:
They allow you to set up the client. https://offix.dev/docs/getting-started#using-the-client-inside-your-application

"apollo-cache-inmemory is the recommended cache implementation for Apollo Client 2.0. InMemoryCache is a normalized data store that supports all of Apollo Client 1.0's features without the dependency on Redux."

Reply:
thanks! looks like we're using it in #63 as well so hopefully that makes things easier to merge and resolve conflicts